### PR TITLE
link to google captcha service

### PR DESCRIPTION
**Pull request by cbeauchesne**

### CODE REVIEW

File: src/views/user/LoginView.vue [hunk 1]
```diff
@@ -216,6 +256,13 @@
             })
         },
 
+        mounted() {
+            let recaptchaScript = document.createElement('script')
```
Comment (brunobesson) [inline src/views/user/LoginView.vue]: could be a `const` (but don't correct this now, sometime soon we'll have auto linting for this)

File: vue.config.js [hunk 2]
```diff
@@ -52,15 +52,17 @@ const config = {
...
         },
         prod: {
             name: 'prod',
             api: 'https://api.camptocamp.org',
             media: 'https://media.camptocamp.org/c2corg_active',
             imageBackend: 'https://images.camptocamp.org',
             forum: 'https://forum.camptocamp.org',
-            readWrite: false // conservative approach : you have to explicity set it to true
+            readWrite: false, // conservative approach : you have to explicity set it to true
+            recaptchaKey: '6Lc9Cw4UAAAAAIKnlar0AOsGX_P5S-bk9u8viuo2'
```
Comment (brunobesson) [inline vue.config.js]: is it OK to store this key on the repo? it doesn't need to be hidden?

Comment (cbeauchesne) [reply vue.config.js]: yes, it's the public key send from UI to google server (that the way I've getted them :) )

File: src/views/user/LoginView.vue (hunk 1 above)
Comment (cbeauchesne) [reply src/views/user/LoginView.vue]: 👍

**Pull request merged**